apps/src/templates/curriculumCatalog/curriculum_catalog_card.module.scss

```diff
@@ -133,6 +153,8 @@ $container-width: 309px;
   a:link,
   a:visited {
     width: 100%;
+    display: flex;
+    justify-content: center;
```

Comment:
This fixes the bug in which the text was not centered on international versions

Reply:
Nice catch! Thank you!